pyproject.toml

```diff
@@ -45,8 +45,9 @@ use_parentheses = false
 [tool.pyright]
 pythonVersion = "3.11"
 
-[tool.pdm]
-version = { source = "scm" }
+[tool.pdm.version]
+source = "file"
+path = "registry/version.py"
```

Comment:
Since we aren't doing any fancy versioning for this repo, consider using a git tag to derive the version. I will approve as-is since you are more familiar with the logistics of incrementing versions for various releases.